test/compare/default/obj_expression-in.js

```diff
@@ -85,8 +85,9 @@ Ext.define('VMS.model.User', {
     {
       name: 'LastName',
       type: 'string'
+      // align with previous line because of line break
 
-      // lorem ipsum
+      // align with "}"
```

Comment:
If `align with "}"` applied, shouldn't this comment and the two below have less indent?